lib/validators.js

```diff
@@ -8,7 +8,7 @@
  * Regex for valid IDs
  * @type {RegExp}
  */
-var rValidId = /^(?:[a-z0-9]+(?:-[a-z0-9]+)*)$/;
+const rValidId = /^(?:[a-z0-9]+(?:-[a-z0-9]+)*)$/u;
```

Comment:
This is the only violation that was not autofixed (I fixed manually).

https://eslint.org/docs/rules/require-unicode-regexp